nengo_ocl/tests/conftest.py

```diff
@@ -4,24 +4,11 @@
 import pyopencl as cl
 import pytest
 
-import nengo_ocl
-
 from nengo.conftest import plt, rng  # noqa: F401
 
 
-# --- Ensure all Simulators have same context
 ctx = cl.create_some_context()
-
-
-class OclSimulator(nengo_ocl.Simulator):
-    def __init__(self, *args, **kwargs):
-        super(OclSimulator, self).__init__(*args, context=ctx, **kwargs)
-
-
 @pytest.fixture(scope="session")
```

Comment:
Sorry, think this was a messed up `git add -p`. I'll run through it again and fix up.